### PR TITLE
Feat: APP-2230 tweaking dao creation flow to account for wrapped tokens

### PR DESCRIPTION
**Pull request by babarella**
## Description

Tweaking dao creation flow to account for wrapped tokens

Task: [APP-2230](https://aragonassociation.atlassian.net/browse/APP-2230) [APP-2226)](https://aragonassociation.atlassian.net/browse/APP-2226)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist:
- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2230]: https://aragonassociation.atlassian.net/browse/APP-2230?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/web-app/src/containers/configureCommunity/minParticipation.tsx [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+    name: [
+      'tokenTotalSupply',
+      'tokenSymbol',
+      'tokenType',
+      'isCustomToken',
+      'minimumParticipation',
+    ],
+  });
+
+  const isGovTokenRequiresWrapping =
```
Comment (sepehr2github) [inline packages/web-app/src/containers/configureCommunity/minParticipation.tsx]: The min participation flow should be applied only for ERC20 and not ERC20 governance

File: packages/web-app/src/containers/setupCommunity/addExistingToken.tsx [hunk 2]
```diff
@@ -129,6 +126,17 @@ const AddExistingToken: React.FC = () => {
...
       </FormItem>
+      {isAllowedToConfigureVotingEligibility && (
+        <FormItem>
+          <DescriptionContainer>
+            <Label
+              label={t('labels.proposalCreation')}
+              helpText={t('createDAO.step3.proposalCreationHelpertext')}
+            />
+          </DescriptionContainer>
+          <SelectEligibility />
```
Comment (sepehr2github) [inline packages/web-app/src/containers/setupCommunity/addExistingToken.tsx:145]: We have an eligibility token component in verification card as well, please merge those duplicated components

**Review by sepehr2github — CHANGES_REQUESTED** [review]
Screenshot 2023-06-25 at 11 22 50 PM

Thanks, Nikita, I think after the merge the flow of ERC20 and governance ERC20 is broken, For example, minimum participation doesn't look proper for compatible token

**Comment by babarella** [comment]
> Screenshot 2023-06-25 at 11 22 50 PM
> Thanks, Nikita, I think after the merge the flow of ERC20 and governance ERC20 is broken, For example, minimum participation doesn't look proper for compatible token

Resolved, thank you for your findings.

**Review by sepehr2github — APPROVED** [review]
Thanks

**Pull request merged**